bundles/org.openhab.binding.sonnen/README.md

```diff
@@ -0,0 +1,58 @@
+# Sonnen Binding
+
+The binding for sonnen communicates with a sonnen battery. More information about the sonnen battery can be found [here](https://sonnen.de/)
+
+## Supported Things
+
+| Things | Description  | Thing Type |
+|--------|--------------|------------|
+| sonnen | Control of a sonnen battery | battery|
+
+
+## Thing Configuration
+
+Only one parameter is required. The IP-Address of the sonnen battery in your local network.
+
+```
+Thing sonnen:battery:myBattery "sonnenBattery"  [ hostIP="192.168.0.10"]
+```
+
+## Channels
+
+The following channels are yet supported:
+
+
+| Channel | Type  | Access| Description|
+|---------|-------|-------|------------|
+|batteryCharging|Switch|read|Indicates if the Battery is charging at that moment|
+|batteryDischarging|Switch|read|Indicates if the Battery is discharging at that moment|
+|batteryFeedIn|Number:Energy|read|Indicates the actual charging current of the Battery in watt|
+|batteryDispense|Number:Energy|read|Indicates the actual current dispense from the Battery in watt|
+|consumption|Number:Energy|read|Indicates the actual consumption of the consumer in watt|
+|gridFeedIn|Number:Energy|read|Indicates the actual feeding to the Grid in watt.0 if nothing is feeded|
+|gridReceive|Number:Energy|read|Indicates the actual receiving from the Grid in watt.0 if nothing is received|
+|solarProduction|Number:Energy|read|Indicates the actual production of the Solar system in watt|
+|batteryLevel|Number|read|Indicates the actual Battery Level in % from 0 - 100|
+|flowConsumptionBattery|Switch|read|Indicates if there is a current flow from Battery towards Consumption|
+|flowConsumptionGrid|Switch|read|Indicates if there is a current flow from Grid towards Consumption|
+|flowConsumptionProduction|Switch|read|Indicates if there is a current flow from Solar Production towards Consumption|
+|flowGridBattery|Switch|read|Indicates if there is a current flow from Grid towards Battery|
+|flowProductionBattery|Switch|read|Indicates if there is a current flow from Production towards Battery|
+|flowProductionGrid|Switch|read|Indicates if there is a current flow from Production towards Grid|
+
+## Full Example
+
+demo.items:
+
+```
+Number:Energy Consumption { channel="battery:consumption" }
+Number:Energy GridFeeding { channel="battery:gridFeedIn" }
+Number BatteryLevel { channel="battery:batteryLevel" }
+Switch FlowConsumptionBattery { channel="oven:flowConsumptionBattery" }
```

Comment:
To be fixed.
I provided the exact syntax in a previous comment.
Of course, here again, the channel id will depend on your final channel id naming.
`Number:Energy Consumption { channel="sonnen:battery:myBattery :Consumption" }`